src/utils/helpers/attemptLogin.ts

```diff
@@ -41,17 +43,14 @@ export const attemptLogin = async ({
 export const attemptLoginOrg = async ({
   orgId,
   sessionId,
-  JWT,
   onLoginOrgSuccess,
   onLoginOrgFailure = () => {},
 }: loginOrgParameters) => {
   try {
-    const authHeader = { Authorization: 'Bearer ' + JWT }
```

Comment:
Just doing testing and updates on this now. Problem with removing JWT from here and relying on localStorage in the "POST" method is that the initial Login user process didn't set JWT in local storage yet.

I could make it do so, but there could still be a timing issue if loginOrg runs before JWT is written to localStorage. I think it would be better if the GET/POST functions allowed a JWT/auth parameter like they did before and only check the localStorage if not provided, so we can use that mechanism for this initial login and login org.

Reply:
It works if I write to localStorage straight after initial login. Hopefully it won't encounter a situation where the "onLoginOrg" method runs before it's properly in localStorage.